src/DeepCopy/Filter/ReplaceFilter.php

```diff
@@ -1,18 +1,12 @@
-<?php
+<?php declare(strict_types=1);
 
 namespace DeepCopy\Filter;
 
 use ReflectionProperty;
 
-/**
- * @final
- */
-class ReplaceFilter implements Filter
+final class ReplaceFilter implements Filter
 {
-    /**
-     * @var callable
```

Comment:
I am against this removal. A lot of IDE and SCA tools can mess.

Reply:
The typehint is provided in the constructor, since it's a very straightforward case (single point of injection, no conditional assignment) any decent IDE or static analysis should be able to pick it up. And the gain is that it removes unnecessary noise and reduce the risk of having a docblock unsync with the underlying code

Reply:
:+1: